specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/examples/Diagnostics_GetSiteDetectorResponse.json

```diff
@@ -0,0 +1,137 @@
+{
+    "parameters": {
+        "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+        "resourceGroupName": "Sample-WestUSResourceGroup",
+        "siteName": "SampleApp",
+        "detectorName": "runtimeavailability",
+        "api-version": "2016-03-01"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/ef90e930-9d7f-4a60-8a99-748e0eea69de/resourceGroups/Build2015DemoRG/providers/Microsoft.Web/sites/BuggyBakery/detectors/runtimeavailability",
+                "name": "runtimeavailability",
+                "location": "West US",
```

Comment:
"location" is not allowed for a proxy resource.

Reply:
Removed